### ISSUE TITLE
"PATH" has to end with a slash

### ISSUE THREAD

**Issue by rugk** [PrivateBin/PrivateBin#86]
## Steps to reproduce

1. Change https://github.com/PrivateBin/PrivateBin/blob/master/index.php#L14 to a path not ending with a slash. (`../../somewhereelse`)

### What happens
Server errors.

### What should happen
The path should either be appended internally or we should clearly state in the comment above that the path has to end with a slash.

## Basic information

**PrivateBin version:** 0.22

**Comment by elrido** [comment]
Lets take this issue and the other about the server salt exception as an opportunity to add some warning messages for server admins.

**Comment by rugk** [comment]
Well, also the comment should say this to the admin.

**Comment by elrido** [comment]
The server salt one already throws a quite clear Exception, so I did only clarify the installation documentation (wiki and INSTALL.md) on this point.

As the PATH is a constant, we can not easily autofix it and I did not want to introduce lots of such checks & corrections for such a configuration issue everywhere we use the PATH. Instead it now throws an exception with clear instructions on how to fix it.
